pkg/karmadactl/addons/init/enable_option.go

```diff
@@ -0,0 +1,117 @@
+package init
+
+import (
+	"context"
+	"fmt"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	"k8s.io/utils/strings/slices"
+
+	cmdinit "github.com/karmada-io/karmada/pkg/karmadactl/cmdinit/kubernetes"
+	"github.com/karmada-io/karmada/pkg/karmadactl/cmdinit/utils"
+)
+
+// CommandAddonsEnableOption options for addons list.
+type CommandAddonsEnableOption struct {
+	GlobalCommandOptions
+
+	KarmadaSearchImage string
+
+	KarmadaSearchReplicas int32
+
+	KarmadaDeschedulerImage string
+
+	KarmadaDeschedulerReplicas int32
+
+	KarmadaEstimatorImage string
+
+	KarmadaEstimatorReplicas int32
+
+	KarmadaKubeClientSet *kubernetes.Clientset
+
+	WaitPodReadyTimeout int
+
+	WaitAPIServiceReadyTimeout int
+
+	MemberKubeConfig string
+
+	MemberContext string
+}
+
+// Complete the conditions required to be able to run enable.
+func (o *CommandAddonsEnableOption) Complete() error {
+	err := o.GlobalCommandOptions.Complete()
+	if err != nil {
+		return err
+	}
+
+	o.KarmadaKubeClientSet, err = utils.NewClientSet(o.KarmadaRestConfig)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Validate Check that there are enough conditions to run addon enable.
+func (o *CommandAddonsEnableOption) Validate(args []string) error {
```

Comment:
Suggest to check if the args is valid, because there is no prompt when the name of addons is error.

```
[root@master67 karmada]# ./karmadactl addons enable ss --kubeconfig /root/.kube/config --context karmada-host --karmada-kubeconfig /root/.kube/config --karmada-context karmada-apiserver -v=6
I0726 11:27:20.533215    3062 loader.go:372] Config loaded from file:  /root/.kube/config
I0726 11:27:20.539797    3062 loader.go:372] Config loaded from file:  /root/.kube/config
I0726 11:27:20.573281    3062 round_trippers.go:553] GET https://10.10.103.67:6443/api/v1/namespaces/karmada-system/secrets/kubeconfig 200 OK in 31 milliseconds
[root@master67 karmada]#
```